Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -2215,6 +2293,9 @@ namespace Babylon
             NativeWebXRRenderTarget::Initialize(env);
             NativeRenderTargetProvider::Initialize(env);
             XR::Initialize(env);
+
+            // XRHand needs to be defined in order to use hand tracking, but we don't actually need the interface
+            env.Global().Set("XRHand", true);
         }
```

Comment:
nit: its probably worth having an XRHand interface for consistency. long term i am pretty sure XRHand is supposed to be iteratable/may need a constructro compared to us being able to get by just setting XRHand to a boolean

Reply:
I was tempted to make an XRHand component, but after talking with @syntheticmagus we decided that, for now at least, we can get by with a boolean, since implementing XRHand here as a wrapped class (and an iterable one, at that) was a lot more work. More so since we can achieve the same result for now by creating an Napi::Array, and do so on line 179. We can always replace it later if needed.